### PR TITLE
Add index.js defining main module exports

### PR DESCRIPTION
**Pull request by saintedlama**
Adds index.js that defines main module exports `Inspector` and `reporters`. Makes PR #41 obsolete

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  Inspector: require('./lib/inspector'),
+  reporters: require('./lib/reporters')
+};
```
Comment (danielstjules) [inline index.js]: Newline? :)

**Comment by danielstjules** [comment]
Thanks again!

File: index.js (hunk 1 above)
Comment (saintedlama) [inline index.js]: Newline?

Comment (danielstjules) [inline index.js]: Oh, can you add a new line at the end of the file?

**Pull request merged**